clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1452,9 +1460,15 @@ instruction information to go along with each load and store, while cache
 simulators can ignore these "no-fetch" entries and avoid incorrectly
 inflating instruction fetch statistics.
 
-Traces include scheduling markers providing the timestamp and hardware
-thread identifier on each thread transition, allowing a simulator to more
-closely match the actual hardware if so desired.
+Traces include scheduling markers providing the timestamp and hardware thread identifier
+on each thread transition, allowing a simulator to more closely match the actual
+hardware if so desired: but be aware that this "as-traced" schedule is not
+representative, as shown in \ref sec_drcachesim_as_traced.  We recommend instead using
+dynamic re-scheduling of the software threads: see \ref sec_drcachesim_sched_dynamic.
+While we suggest keeping traces stored as thread-sharded and using the dynamic scheduler
+in each run, there is support for running the scheduler once and creating a new set of
+stored traces in core-sharded format: essentially switching to hardware-thread-oriented
+traces.  This is done using the \ref sec_tool_record_filter tool in `-core_sharded` mode.
```

Comment:
This might be a good place to mention whether the scheduling markers included in the trace are updated, removed, or left as-is when traces are rescheduled.